runtime/Cargo.toml

```diff
@@ -88,6 +88,7 @@ deno_webidl.workspace = true
 deno_websocket.workspace = true
 deno_webstorage.workspace = true
 fastwebsockets.workspace = true
+rustyline = { version = "=10.0.0", default-features = false, features = ["custom-bindings"] }
```

Comment:
Please move this line to `Cargo.toml` in the root of the repo and use `rustyline.workspace = true`. Similarly `cli/Cargo.toml` will need to be updated with `rustyline.workspace = true`